### PR TITLE
feat: minor reorder SQL Lab Tab controls

### PR DESCRIPTION
**Pull request by graceguo-supercat**
### SUMMARY
A few airbnb users reported that they accidentally clicked status icon, and they lost the active query.
This PR is to re-order the controls in each SQL Lab tab to prevent such accidents.

- re-order dropdown menu and tab title
- separate the status icon and close button
- always show close button at the right-side

Thanks for @kenchendesign brought up design!

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
**Before:**
Screen Shot 2020-07-08 at 10 56 39 AM

**After:**
Screen Shot 2020-07-08 at 9 36 37 AM

### TEST PLAN
Integration test and manual test.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [X] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by ktmud** [comment]
image

Maybe separate to this, can we add a close icon for table preview tabs as well?

**Comment by graceguo-supercat** [comment]
- it's totally separated component and functions.
- you can close preview from control panel:

L1rDIBpWyL

**Comment by rusackas** [comment]
I'm in total agreement that the status dot acting as a close button is a dangerous pattern.

If closing the tab is a risk, would it be reasonable to remove the X button from the new design as well, and require that users click the ⠇menu and select "close tab" to close the tab?

If that's reasonable, I think the menu (whether a ⠇or a ˅ ) would be better back on the right side of the tab. I'm a bit ambivalent about whether the status dot should be to the left or right of the title.

**In other words** would it make sense to just remove the "X" close button from the status dot and call it a day?

**Comment by ktmud** [comment]
Yeah, I know that close button exists. Just a suggestion. Haven't used SQL Lab for a while and was surprised yesterday that it's not obvious how to close the preview tab.


> **In other words** would it make sense to just remove the "X" close button from the status dot and call it a day?

+1 to this, was just typing the same thing.

**Comment by mistercrunch** [comment]
Generally it's probably best to align with the way browser tabs behave as the most commonly used tabs out there.

Visually the `x` is a bit bold and large. Maybe we can mute it a little (?) - sorry to ask for CSS adjustments on a PR... Could also only show the `x` for the active tab to make things more busy (?)

Stepping back, I think the goal longer term is still to get rid of in-app tabs and push users to use browser tabs instead.

**Comment by graceguo-supercat** [comment]
I like to keep `x` as short-cut in the sql lab tab...especially when you want to close many tabs. click drop-down then click close need a lot more...work :)

**Comment by kenchendesign** [comment]
+1 on making the `x` slightly thinner, I don't think they should be lighter-colored though.

+1 on making all tabs visually consistent with an `x` on the right end.

**Comment by graceguo-supercat** [comment]
Sorry i clicked merge button accidentally. How can it get merged without code review approval?

**Comment by graceguo-supercat** [comment]
So let's discuss how to improve it and I will make another PR for additional comments. Thanks!
